src/nspawn/nspawn-mount.c

```diff
@@ -766,6 +766,12 @@ static int mount_bind(const char *dest, CustomMount *m) {
                         return log_error_errno(r, "Read-only bind mount failed: %m");
         }
 
+        if (uid_shift != 0 && uid_range != 0) {
```

Comment:
```uid_shift != UID_INVALID && uid_range != UID_INVALID```

Reply:
Uhm, UID_INVALID would mean UINT32_C(0xFFFFFFFF) and be something very differently. Or am I missing something here?

Reply:
That's what UID variables are initialized to when they are not set

Reply:
use uid_is_valid() actually.

Reply:
obsolet since it is no handled by a mount option